### PR TITLE
Add meeting notes from April 2020

### LINKED ISSUE #273: Team Meeting April 2020
Hello everyone, 

Reminder that our next meeting is **Thursday, April 16th, 2020** at 8AM UTC (or your [timezone](https://arewemeetingyet.com/UTC/2020-04-16/08:00/JupyterHub%20and%20BinderHub%20Team%20Meeting)). 

* You can find the agenda [here](https://hackmd.io/u2ghJJUCRWK-zRidCFid_Q?both).
* And we'll meet on Google Hangouts (video conferencing) [here](https://meet.google.com/wat-vtby-cfd)

These calls are open to **anyone**, so please join us! We love seeing new members on the call.

### PR DESCRIPTION
**Pull request by sgibson91**
fix #273

### CODE REVIEW

File: docs/monthly-meeting/2020-04-16.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+        * Tim thinks starting on a small number of repos is good, then we can experiment and maybe at the next call decide to roll it out to all repos?
+    * Q: what are your thoughts on a PR template as well? For example https://github.com/alan-turing-institute/the-turing-way/blob/master/.github/PULL_REQUEST_TEMPLATE.md
+    * Todo: open a PR on team-compass to document this process to help spread the word and get a consistent response from all maintainers
+
+* Increasing limits for "special" repositories [Tim]
+    * https://github.com/jupyterhub/team-compass/issues/276
+    * We increased the limit for the repo mentioned in the issue to 200 for the duration of a live stream
+    * do we need a bigger process around increasing the limit for a repository for a short amount of time? what response time?
+    * Does it matter who is asking for the increase? Paid vs free vs one off vs "evey week"?
```
Comment (manics) [inline docs/monthly-meeting/2020-04-16.md:53]:
```suggestion
    * Does it matter who is asking for the increase? Paid vs free vs one off vs "every week"?
```

File: docs/monthly-meeting/2020-04-16.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    * in the context of https://github.com/jupyterhub/team-compass/issues/276 a discussion around funding for mybinder.org started
+    * There is not a lot of structure to this discussion item. Maybe if we have some spare time at the end of the meeting we can chat about it.
+    * [name=Simon] Would direct sponsorship from a conference imply a Service Level Agreement? Alternative could be to promote Sarah's one-click Azure deployment?
+    * [name=Erik] If we take payments, it must be as simple as possible to manage because we don't have resources to manage this afaik.
+    * Q: can people directly buy us credits on GKE/AWS/Azure?
+    * Q: why do we need money?  A: We are always looking for cloud credits.
+    * One way to help us is to use your influence to make it easier for Binder to continue to get credits from those that give them to us (GKE, Gesis management)
+    * Accepting money ourselves is a *huge* amount of work and {possibly,probably} not worth it.
+    * Demonstrate the value we deliver!
+        * maybe a exploration of our binder analytics archive
```
Comment (manics) [inline docs/monthly-meeting/2020-04-16.md:75]:
```suggestion
        * maybe an exploration of our binder analytics archive
```

File: docs/monthly-meeting/2020-04-16.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    * [name=Simon] Helm 3 works well for new deployments of JupyterHub and BinderHub (I've been using it). It *won't* upgrade from Helm 2 deployments:
+        * https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/
+        * https://helm.sh/docs/topics/v2_v3_migration/
+    * [name=Erik] General notes on helm2 and helm3 upgrades:
+        * Always use `--cleanup-on-fail`.
+        * Don't use `--wait`, or if you do, ensure to not abort!
+        * Additional notes: https://github.com/consideRatio/consideratio.github.io/issues/6
+    * [name=Erik] I've successfully upgraded my old Helm2 JupyterHub deployment, and now use Helm3. I used the helm2to3 plugin.
+    * [name=Erik] Relevant understanding:
+        * Helm store's information about what it has deployed both with Helm2 and Helm3 in Kubernetes, but this "release information" is moved from configmaps in kube-system to secrets in the namespace where the Helm chart is installed.
```
Comment (manics) [inline docs/monthly-meeting/2020-04-16.md:93]:
```suggestion
        * Helm stores information about what it has deployed both with Helm2 and Helm3 in Kubernetes, but this "release information" is moved from configmaps in kube-system to secrets in the namespace where the Helm chart is installed.
```

**Comment by consideRatio** [comment]
@sgibson91 @manics :heart: :+1:

**Pull request merged**